api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -1272,6 +1273,21 @@ public static Properties getConfigProperties() {
 	public static void updateSearchIndex() {
 		getContextDAO().updateSearchIndex();
 	}
+
+	/**
+	 * Updates the search index. It is an asynchronous operation.
+	 * <p>
+	 * There is no need to call this method in normal usage since the index is automatically updated
+	 * whenever DB transactions are committed.
+	 * <p>
+	 * The method is designated to be used in tests, which rollback transactions. Note that if the
+	 * transaction is rolled back, changes to the index will not be reverted.
```

Comment:
Could you please fix the javadoc? It is not intended for tests rather for use by modules to update the index in background after any database changes done omitting OpenMRS API and Hibernate.